spec/services/cfe_civil/components/employments_spec.rb

```diff
@@ -9,7 +9,7 @@
     before { create(:applicant, :not_employed) }
 
     it "renders the expected, empty hash" do
-      expect(call).to eq({})
+      expect(call).to eq({}.to_json)
```

Comment:
could just use "{}" i believe, but either way

Reply:
That seemed like it would be less clear in six months! 😄